scene/gui/code_edit.cpp

```diff
@@ -3344,5 +3342,5 @@ bool CodeCompletionOptionCompare::operator()(const ScriptLanguage::CodeCompletio
 			return l.matches[i].second > r.matches[i].second;
 		}
 	}
-	return l.display < r.display;
+	return l.display.to_lower() < r.display.to_lower();
```

Comment:
```suggestion
	return l.display.naturalnocasecmp_to(r.display) < 0;
```
Or better yet, to follow the way documentation works:
```suggestion
	return l.display.naturalcasecmp_to(r.display) < 0;
```